diff --git a/tests/signtest.cpp b/tests/signtest.cpp
index 80e2c025..d13a518b 100644
--- a/tests/signtest.cpp
+++ b/tests/signtest.cpp
@@ -21,7 +21,7 @@
 
 // On Centos 7, remquo(810.0, 90.0 &q) returns 90.0 with q=8.  Rather than
 // lousing up Math.cpp with this problem we just skip the failing tests.
-#if defined(__GNUG__) && __GNUG__ < 11
+#if 1
 #  define BUGGY_REMQUO 1
 #else
 #  define BUGGY_REMQUO 0
