diff --git a/tests/signtest.cpp b/tests/signtest.cpp
index 80e2c025..f81501a9 100644
--- a/tests/signtest.cpp
+++ b/tests/signtest.cpp
@@ -21,7 +21,7 @@
 
 // On Centos 7, remquo(810.0, 90.0 &q) returns 90.0 with q=8.  Rather than
 // lousing up Math.cpp with this problem we just skip the failing tests.
-#if defined(__GNUG__) && __GNUG__ < 11
+#if !defined(_MSC_VER)
 #  define BUGGY_REMQUO 1
 #else
 #  define BUGGY_REMQUO 0
@@ -37,7 +37,7 @@
 // Broken on build-open machines, vc14 and vc15+win32
 // Fixed on build-open machines, vc15+x64 and vc16
 // Let's assume that it's OK for vc16 and later
-#if defined(_MSC_VER) && _MSC_VER < 1920
+#if defined(_MSC_VER)
 #  define BUGGY_ROUNDING 1
 #else
 #  define BUGGY_ROUNDING 0
